cmd/kube-apiserver/app/server.go

```diff
@@ -111,6 +111,7 @@ type APIServer struct {
 	SSHUser                    string
 	SSHKeyfile                 string
 	MaxConnectionBytesPerSec   int64
+	ApiServerNodePort          int
```

Comment:
it actually doesn't interact with ServiceNodePortRange. Ill see what a reasonable idiom is for this and update.